docs/integrations/sources/klaviyo-migrations.md

```diff
@@ -0,0 +1,6 @@
+# Klaviyo Migration Guide
+
+## Upgrading to 1.0.0
+
+`event_properties/items/quantity` for `Events` stream is changed from `integer` to `number`.
```

Comment:
Given that an integer is a subset of a number shouldn't users be able to continue with their syncs?

Im sure im missing something 😅

Reply:
Just unresolving this because im still not positive if this should be a breaking change or not?

Reply:
Actually, we move from integer to number.
we already had one customer who tries to sync with the current schema (which expects integer) but received error on the destination because value was 4.0





[12:32](https://airbyte-globallogic.slack.com/archives/D061ZSGPPV1/p1697837545035799)
So field for  destination table should be update from integer to number to be able to store 4.0